### PR TITLE
Cleanup untagged nightly images

### LINKED ISSUE #378: The cleanup ACR images pipeline should be cleaning up the nightly repos.
Our build ACR is ever increasing in size.  We should have cleanup logic for the public nightly repos.  There is no reason to keep every image we produce.  I think it would be reasonable to delete all untagged images from these repos which should dramatically reduce the size of the repos.

### PR DESCRIPTION
**Pull request by mthalman**
Updates the script to cleanup ACR by including untagged images in the nightly repos.

Fixes #378

### CODE REVIEW

File: eng/Invoke-AcrCleanup.ps1 [hunk 1]
```diff
@@ -27,22 +45,43 @@ az account set --subscription $SubscriptionId
...
+
+            $deletedRepos += $repoName
         }
+    }
+    else {
+        Write-Host "Querying manifests"
+        $manifests = az acr repository show-manifests --name $RegistryName --repository $repoName | ConvertFrom-Json
+
+        # Delete all of the untagged images
+        $untaggedImages = $manifests | Where-Object { $_.tags.Count -eq 0 }
```
Comment (MichaelSimons) [inline eng/Invoke-AcrCleanup.ps1]: Should this also include a date component?  e.g. delete all untagged images older than expiration period used in the staging repos cleanup logic.

File: eng/Invoke-AcrCleanup.ps1 [hunk 2]
```diff
@@ -57,5 +96,12 @@ foreach ($deletedRepo in $deletedRepos) {
 }
 
 Write-Host ""
+Write-Host "Deleted images:"
+foreach ($deletedImage in $deletedImages) {
+    Write-Host $deletedImage
```
Comment (MichaelSimons) [inline eng/Invoke-AcrCleanup.ps1]: Does this display information about what the parent repo is?  I think this should be displayed.

Comment (mthalman) [reply eng/Invoke-AcrCleanup.ps1]: Yes, an example is `public/dotnet/core-nightly/sdk@sha256:fff9f9ae1e00b84421588c37650aa7109e4c95d222007f2c72ad00d8a6409830`

File: eng/Invoke-AcrCleanup.ps1 (hunk 1 above)
Comment (mthalman) [reply eng/Invoke-AcrCleanup.ps1]: Good point

**Pull request merged**